emsdk

```diff
@@ -2531,13 +2531,17 @@ def main():
 
     return 0
   elif cmd == 'construct_env':
-    silentremove(EMSDK_SET_ENV) # Clean up old temp file up front, in case of failure later before we get to write out the new one.
+    if len(sys.argv) == 2:
+      outfile = EMSDK_SET_ENV
+      silentremove(EMSDK_SET_ENV) # Clean up old temp file up front, in case of failure later before we get to write out the new one.
+    else:
+      outfile = sys.argv [2]
```

Comment:
should not be a space between `argv` and `[`

Reply:
Fixed.

Reply:
Perfect, thanks!